examples/keyword.rs

```diff
@@ -0,0 +1,21 @@
+/// Demostrates how to fetch and set keywords
+/// 
+/// Usage: cargo run --example keyword <keyword> <value>
+/// Example: cargo run --example keyword decoration:rounding (prints value) 
+/// Example: cargo run --example keyword decoration:rounding  15 (sets value)
+
+use hyprland::keyword::Keyword;
+
+fn main() -> hyprland::Result<()> {
+    let args: Vec<_> = std::env::args().skip(1).collect();
+    let keyword = if args.len() == 0 { "general:border_size" } else { &args[0].as_str() };
```

Comment:
please dont have a default, and instead exit the process with an error

Reply:
Done